apps/docs/mint.json

```diff
@@ -141,10 +141,8 @@
               "pages": [
                 "libraries/ts/sdk/keys/create",
                 "libraries/ts/sdk/keys/verify",
-                "libraries/ts/sdk/keys/get",
-                "libraries/ts/sdk/keys/verifications",
                 "libraries/ts/sdk/keys/update",
-                "libraries/ts/sdk/keys/delete"
+                "libraries/ts/sdk/keys/revoke"
```

Comment:
This should be `libraries/ts/sdk/keys/delete`

Reply:
changed